CONTRIBUTOR_LADDER.md

```diff
@@ -0,0 +1,81 @@
+    * Must have successful contributions to the project, including at least one of the following:
+        * Accepted and/or reviewed PRs,
+        * Made contributions that resolved Issues
+        * Or some equivalent combination or contribution
+    * Must be actively contributing to at least one subproject, where the privileges of Organization Member will be beneficial
+* Privileges:
+    * May be assigned issues and reviews
+    * Can trigger CI
+    * Can drive project releases
+    * Can be added as code owner for sub-projects
+    * Can recommend other contributors to become Notary Project Organization Members
+
+The process for a Contributor to become an Organization Member is as follows:
+
+1. Any organization member can open an issue in the repo [.github](https://github.com/notaryproject/.github) to nominate a new organization member.
+2. Receive "LGTM" or "+1" comments from a 50% majority of Org maintainers.
+
+## Subproject Maintainer
+
+Subproject Maintainer is defined in [Notary Project governance document](GOVERNANCE.md#subproject-maintainers).
+
+## Org Maintainer
+
+Org Maintainer is defined in [Notary Project governance document](GOVERNANCE.md#notary-project-org-maintainers).
+
+## Inactive Member
+
+_Members are continuously active contributors in the community._
+
+A core principle in maintaining a healthy community is encouraging active participation. It is inevitable that people's focuses will change over time and they are not expected to be actively contributing forever.
+
+However, being a member of one of the Notary Project organizations comes with an elevated set of permissions. These capabilities should not be used by those that are not familiar with the current state of the Notary Project.
+
+Therefore members with an extended period away from the project with no activity will be removed from the Notary Project organizations and will be required to go through the org membership process again after re-familiarizing themselves with the current state. See the Notary Project [governance document](GOVERNANCE.md#maintainers-structure) for the guidance on inactive subproject maintainers and org maintainers.
+
+### How inactivity is measured
+
+Inactive members are defined as members of the Notary Project with **no** contributions across any repository within 12 months. According to the Notary Project [governance document](GOVERNANCE.md), inactive subproject maintainers are defined as members of the Notary Project with **no** contributions in specific subprojects within 6 months. Inactive org maintainers are defined as members of the Notary Project with **no** contributions in specific subprojects within 3 months. This is measured by the CNCF [DevStats project](https://notary.devstats.cncf.io/d/8/dashboards?orgId=1&refresh=15m).
+
+**Note:** Devstats does not take into account non-code contributions. If a non-code contributing member is accidentally removed this way, they may open an issue to quickly be re-instated.
```

Comment:
Since activity includes both repo activity and non-repo activity, e.g., attending meetings and organizing events, perhaps the description of activity in the previous paragraph should explicitly include non-repo activity.